config/initializers/flow_preview.rb

```diff
@@ -2,7 +2,7 @@
 # environment
 show_drafts = ENV["SHOW_DRAFT_FLOWS"] ? true : !Rails.env.production?
 
-FLOW_REGISTRY_OPTIONS = { show_drafts: show_drafts }
+FLOW_REGISTRY_OPTIONS = { show_drafts: show_drafts } # rubocop:disable Style/MutableConstant
```

Comment:
I'm afraid not - this should explain it: 844f75d8ff5d424a81db5a3ced85ccf4023e884c

Reply:
Oh I see, because it references a variable

Maybe changing it to:
```
FLOW_REGISTRY_OPTIONS = {
  show_drafts: ENV.has_key?("SHOW_DRAFT_FLOWS") || !Rails.env.production?
}.freeze
```

Reply:
Ah sorry, that's not what I meant. It's because of things like this: https://github.com/alphagov/smart-answers/blob/master/test/helpers/fixture_flows_helper.rb#L8

Reply:
Oh dear!